tools/shared.py

```diff
@@ -2290,7 +2290,7 @@ def is_wasm_only():
 
   @staticmethod
   def get_safe_internalize():
-    if not Building.can_build_standalone():
+    if Settings.EXPORT_ALL or not Building.can_build_standalone():
       return [] # do not internalize anything
```

Comment:
I think this changes the current meaning of the flag? Docs say

> Note that this does *not* affect LLVM, so it can still eliminate functions as dead. This just exports them on the Module object.

("them" should be "what is left", for clarity). If we do not internalize anything, it won't perform DCE on unneeded things.

Reply:
Oh, I see.  I drop this change then.

I was assuming this option was to tell the linker to keep all the symbols i.e. "EXPORTED_FUNCTIONS=*".